en_US/modules/jwt_authentication.md

```diff
@@ -4,29 +4,36 @@
 
 ## Create module
 
-Open [EMQX Dashboard](http://127.0.0.1:18083/#/modules), click the "Modules" tab on the left, and choose to add:
+Open [EMQX Dashboard](http://127.0.0.1:18083/#/modules), click the "Modules" tab on the left, select "Add Module"
 
-![image-20200928161310952](./assets/modules.png)
+![](./assets/auth_jwt1.png)
 
-Click "Select", and then select the JWT authentication module
+Then select "JWT Authentication" under "Authentication":
 
-![image-20200928141305205](./assets/auth_jwt1.png)
+![](./assets/auth_jwt2.png)
 
-You need to configure the basic connection parameter table such as JWT key, authentication source, and public key (optional).
+JWT authentication provides the following configuration items:
 
-![image-20200928141327836](./assets/auth_jwt2.png)
+1. From. The field that stores the JWT when the client connects, currently supports the selection of username or password.
```

Comment:
Each of these keys should end with a `:` instead of a period. So, for example it should read `From: The...` and `Secret: The...`